src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -205,7 +205,12 @@ internal JObject LoadHostConfig(string configFilePath)
                     }
                     catch (JsonException ex)
                     {
-                        throw new FormatException($"Unable to parse host configuration file '{configFilePath}'.", ex);
+                        var message = $"Unable to parse host configuration file '{configFilePath}'.";
+
+                        DiagnosticEventLoggerExtensions.LogDiagnosticEventError(
```

Comment:
This will only log the inner exception (`ex`). For consistency, it would be good to create the `FormatException`, with the inner exception set as it was before. Log, then throw.

Reply:
Good point. Fixed.